### ISSUE TITLE
Add battery percentage in de bar near the icon

### ISSUE THREAD

**Issue by jcbernard83** [actuallymentor/battery#58]
Add battery percentage in de bar near the icon

**Comment by schabrolles** [comment]
Having more customisation like show the time remaining and/or battery %

**Comment by actuallymentor** [comment]
I'm a bit torn on this. I kind of want to keep this tool very minimalist, especially since many of the commonly asked features are built into the battery icon that the system already has.

Can you explain why you want this feature? Why not just use the built-in battery icon for this?

**Comment by jcbernard83** [comment]
Hi,I want to avoid 2 battery icon in the bar. If battery diplays the percentage I can remove the OS icon. May be you can add too in the menu an item to access to the battery configuration. JC. Le 11 févr. 2023 à 22:25, Mentor Palokaj ***@***.***> a écrit :﻿
I'm a bit torn on this. I kind of want to keep this tool very minimalist, especially since many of the commonly asked features are built into the battery icon that the system already has.
Can you explain why you want this feature? Why not just use the built-in battery icon for this?

**Comment by actuallymentor** [comment]
Is the current dropdown insufficient for you? It shows very similarly to the system dialogue:

<img width="314" alt="image" src="https://user-images.githubusercontent.com/9071382/218307603-a30d4ca8-24a5-4112-8d00-9e6782f47e94.png">

<img width="265" alt="image" src="https://user-images.githubusercontent.com/9071382/218307604-56f1fd95-6585-48ac-88fa-5bf0437aabf0.png">

**Comment by schabrolles** [comment]
well ... I also would like to only have one battery icon... I really like having the possibility to get time remaining battery...

**Comment by actuallymentor** [comment]
But it says the time remaining?

**Comment by schabrolles** [comment]
No, the official battery tray icon gives you only battery percentage. While the command battery status can give you time remaining

**Comment by actuallymentor** [comment]
It gives both, I'm very confused what you are talking about

**Comment by jakjakob** [comment]
you can use just the Command-line version, so you'll have only the system icon
